modules/javafx.controls/src/main/java/javafx/scene/chart/CategoryAxis.java

```diff
@@ -68,8 +68,17 @@ public final class CategoryAxis extends Axis<String> {
     // -------------- PRIVATE FIELDS -------------------------------------------
     private List<String> allDataCategories = new ArrayList<>();
     private boolean changeIsLocal = false;
-    /** This is the gap between one category and the next along this axis */
-    private final DoubleProperty firstCategoryPos = new SimpleDoubleProperty(this, "firstCategoryPos", 0);
+
+    /** This is the position of the first category along this axis */
+    private final DoubleProperty firstCategoryPos =
+            new SimpleDoubleProperty(this, "firstCategoryPos", 0) {
```

Comment:
question: shouldn't line 74 be indented 4 spaces instead 8?

also, we probably should update copyright year in all the touched files, like so:
```
 * Copyright (c) 2010, 2024, Oracle and/or its affiliates. All rights reserved.
```

(I'll do a full review tomorrow)